src/desktop/components/record_artwork_view/index.js

```diff
@@ -0,0 +1,16 @@
+import metaphysics from '../../../lib/metaphysics.coffee'
+import mutation from './mutation'
+import CurrentUser from '../../models/current_user.coffee'
```

Comment:
(Just a note that if these files were written in ES6 we could take advantage of absolute paths)

Reply:
I look forward to that future someday!